src/App.js

```diff
@@ -26,31 +26,34 @@ class App extends Component {
 
     this.state = {
       items: [],
-      points: 0,
+      points: 0
     };
 
     // Uncomment this to spawn a single test item
-    //const testItem = this.spawnItem(Date.now());
-    //this.state.items.push(testItem);
+    // const testItem = this.spawnItem(Date.now());
+    // this.state.items.push(testItem);
+    // console.log(this.state);
 
     // Uncomment this to automatically spawn new items
     this.enableSpawner();
 
-    console.log(this.state);
   }
 
-  onItemClicked = () => {
-    // Fill this in!
-  }
+  // WAVE 3 - Event Handler to add one point to total displayed
+  addPoints = () => {
+    this.setState((prevState) => ({
+      points: prevState.points + 1})
+    )
+  };
 
   render() {
     const items = this.state.items.map((item, i) => {
       return <GameItem
+               addPoints={this.addPoints} // Event Handler for Wave 3
```

Comment:
If this is a callback, made that clear in the name! Fewer people will mess up using your code that way.